web/Services/LandlordService.cs

```diff
@@ -218,6 +219,21 @@ await using (var transaction = await _dbContext.Database.BeginTransactionAsync(I
         return ILandlordService.ApproveLandlordResult.Success;
     }
 
+    public async Task UnapproveLandlord(int landlordId)
+    {
+        var landlord = await GetLandlordIfExistsFromId(landlordId);
+        if (landlord is null)
+        {
+            return;
+        }

```

Comment:
Silently doing nothing is usually a bad way to handle errors because you end up with hard to diagnose bugs. So just do `Landlords.Single` and let it throw an exception if it doesn't exist.

Reply:
Oh yeah that's definitely a good idea